src/agent/README.md

```diff
@@ -1,6 +1,6 @@
 # Kata Agent in Rust
 
-This is a rust version of the [`kata-agent`](https://github.com/kata-containers/kata-agent).
+This is a rust version of the [`kata-agent`](https://github.com/kata-containers/kata-containers/src/agent).
```

Comment:
I don't think this change is accurate.
We should be pointing to: https://github.com/kata-containers/agent (the go one) instead of https://github.com/kata-containers/kata-containers/src/agent (the rust one).

Reply:
Good point! I've fixed it. Thanks.